plugins/modules/ilo_redfish_config.py

```diff
@@ -34,24 +35,29 @@
       - Base URI of iLO.
     type: str
   username:
+    required: false
     description:
       - Username for authenticating to iLO.
     type: str
   password:
+    required: false
     description:
       - Password for authenticating to iLO.
     type: str
   auth_token:
+    required: false
     description:
       - Security token for authenticating to iLO.
     type: str
   timeout:
+    required: false
     description:
-      - Timeout in seconds for HTTP requests to iLO.
-    default: 10
+      - Timeout in seconds for URL requests to iLO controller.
+      - The default value for this param is 10 but that is being deprecated
```

Comment:
```suggestion
      - The default value for this param is C(10) but that is being deprecated
```
https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation